support/cas-server-support-pm-webflow/src/main/java/org/apereo/cas/pm/web/flow/PasswordManagementWebflowConfigurer.java

```diff
@@ -65,8 +66,8 @@ protected void doInitialize() {
     }
 
     private void createAccountStatusViewStates(final Flow flow) {
-        val initialAction = getState(flow, CasWebflowConstants.ACTION_ID_INITIAL_FLOW_SETUP, ActionState.class);
-        initialAction.getActionList().add(requestContext -> {
+        val initialAction = getState(flow, CasWebflowConstants.STATE_ID_INITIAL_AUTHN_REQUEST_VALIDATION_CHECK, ActionState.class);
+        initialAction.getEntryActionList().add(requestContext -> {
```

Comment:
This should be:

```
 flow.getStartActionList().add(...
```

Reply:
Have changed code to fit request made by @mmoayyed and updated PR.

Thanks